lib/python/ensembl/brc4/runnable/process_gff3.py

```diff
@@ -460,28 +454,38 @@ def transfer_description(self, gene):
         Transfer the translation product description as well
         """
         allowed_transcript_types = self.param("transcript_types")
+        skip_re = r"^(conserved )?(hypothetical|putative) protein$"
```

Comment:
if this is going to be used a few times in one call of this function, Python suggests to compile the regex to improve performance:
```
skip_re = re.compile(r"^(conserved )?(hypothetical|putative) protein$")
...
                if skip_re.match(description):
```

Reply:
I already had this check somewhere else actually. I've set it up as a better regex, with re.compile.